docs/sources/logql/log_queries.md

```diff
@@ -387,6 +387,30 @@ will get those labels extracted:
 "status" => "200"
 ```
 
+#### syslog
+
+The **syslog** parser can be added using the `| syslog` and will extract values from [RFC5424](https://datatracker.ietf.org/doc/html/rfc5424) formatted syslog lines.
+
+For example, parsing the following log line
```

Comment:
Excellent addition.
These suggestions attempt to make this prose read easier, and in a similar way to the Pattern parser prose.

```suggestion
The **syslog** parser (`| syslog`) extracts values from [RFC5424](https://datatracker.ietf.org/doc/html/rfc5424)-formatted syslog lines.

Consider this syslog log line:
```